### PR TITLE
documentation: add window var to identify

### PR DESCRIPTION
**Pull request by fsouza**
This is useful for applying themes to the window (using capabilities in
neovim 0.5).

### CODE REVIEW

**Comment by hrsh7th** [comment]
lt's the correct solution. Thank you.

File: autoload/compe/documentation.vim [hunk 1]
```diff
@@ -7,6 +7,7 @@ let s:FloatingWindow = vital#compe#import('VS.Vim.Window.FloatingWindow')
 let s:window = s:FloatingWindow.new()
 call s:window.set_var('&wrap', 1)
 call s:window.set_var('&conceallevel', 2)
+call s:window.set_var('nvim_compe_documentation', 1)
```
Comment (hrsh7th) [inline autoload/compe/documentation.vim:10]: I think `nvim_` prefix should be removed.

Comment (fsouza) [reply autoload/compe/documentation.vim]: Thank you for the quick review!

**Comment by hrsh7th** [comment]
LGTM. Thank you very much!

**Pull request merged**